nebula/ui/components/MZBottomSheet.qml

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import components 0.1
+import Mozilla.Shared 1.0
+
+Drawer {
+    id: drawer
+
+    readonly property int maxHeight: (Qt.platform.os === "ios" ? window.safeContentHeight : window.height) -  MZTheme.theme.sheetTopMargin
+
+    implicitWidth: window.width
+    implicitHeight: maxHeight
+
+    topPadding: 0
+
+    dragMargin: 0
```

Comment:
Should this constant be moved into theme.js? Are we deliberately not using the standard  MZTheme.theme.cornerRadius value here?